Helpers/MediaLaunchObject.h

```diff
@@ -0,0 +1,36 @@
+//
+//  MediaLaunchObject.h
+//  ConnectSDK
+//
+//  Created by Ibrahim Adnan on 1/19/15.
+//  Copyright (c) 2015 LG Electronics. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "MediaControl.h"
+#import "PlayListControl.h"
+#import "LaunchSession.h"
+
+@interface MediaLaunchObject : NSObject
+
+/*! MediaControl object of Media player*/
+@property (nonatomic, strong) id<MediaControl> mediaControl;
+
+/*! PlayList Control Object of Media player*/
+@property (nonatomic, strong) id<PlayListControl> playListControl;
+
+/*! Launch Session object of Media player*/
+@property (nonatomic, strong) LaunchSession *session;
+
+
+/*
+ * Creates an instance of MediaLaunch with given property values.
+ *
+ * @param launchSession LaunchSession to allow closing this media player
+ * @param mediaControl MediaControl object used to control playback
+ * @param playListControl PlayListControl object used to control playlist
+ */
+- (instancetype) initLaunchSession:(LaunchSession *)session withMediaControl:(id<MediaControl>)mediaControl;
```

Comment:
A better name is `-initWithLaunchSession:andMediaControl:`, because you're not initializing LaunchSession here.